src/pages/home/report/ReportActionsView.js

```diff
@@ -145,6 +165,33 @@ class ReportActionsView extends React.Component {
         this.setState({refetchNeeded});
     }
 
+    /**
+     * Checks if the unreadActionIndicator should be shown.
+     * If it does, starts a timeout for the fading out animation and creates
+     * a flag to not show it again if the report is still open
+     */
+    setUpUnreadActionIndicator() {
+        if (!this.props.isActiveReport || !this.shouldShowNewActionIndicator) {
+            return;
+        }
+
+        this.unreadActionCount = this.props.report.unreadActionCount;
+
+        if (this.unreadActionCount > 0) {
+            this.unreadIndicatorOpacity = new Animated.Value(1);
+            this.timers.push(setTimeout(() => {
+                Animated.timing(this.unreadIndicatorOpacity, {
+                    toValue: 0,
+                    duration: 500,
```

Comment:
I think `duration: 500` and `easing: Easing.ease` are both default values, so they can be omitted

Reply:
Really didn't think of looking what the defaults were.
The duration really is `500`, the easing is actually `Easing.inOut(Easing.ease)`. I cannot visually tell the difference between them in such a short and simple animation, good eyes might even find this curve better.
I'll leave the default one.
Thanks for pointing that out.